contents/tutorials/power-users.md

```diff
@@ -0,0 +1,72 @@
+
+Building a better product for power users keeps them happy and helps prevent them from churning. It helps you build a competitive advantage by growing lock-in with these users. It also  encourages average users to increase their usage and become power users.
+
+To create a great product for power users, you must know who they are and how they are using your product. In this tutorial, we’ll cover how to identify and analyze power users. All you need to start is a PostHog instance (and some users).
+
+## Defining a power user
+
+The first step to identifying and analyzing a power user is defining them. There is no one-size-fits-all, every company and product has its own definition. Defining what they do is key to identifying and analyzing them. Ideally, your power users should align with your ideal customer profile and your key product metrics.
+
+One common aspect between definitions of power users is high usage, whether that is visiting every day, using feature lots, or long session times. A power user relies on the product, it is core to their workflow. They also use advanced features more than average.
+
+To help define exactly what a power user looks like, you can use PostHog. To do so, identify the key features for power users and analyze the events and actions related to those features. Some ideas:
+
+- Compare the retention percentage or returning amounts. Power users have higher retention.
+
+- Compare the stickiness (what events are causing users to come back repeatedly). Power users use features repeatedly.
+
+- Create trends for average user usage vs 90th, 95th, and 99th percentile usage. Power users use features significantly more than average. The larger the gap, the better.
+
+If a feature shows higher retention, repeated use, and a large difference in usage to the average, it is likely valuable to a power user. You can use data related to this feature along with qualitative analysis to solidify the definition of your power user.
+
+## Identifying your power user
+
+Once you’ve decided your definition is a good one, you can use it to create a cohort. To create a power user cohort, we rely on two criteria:
+
+- **Completed an event multiple times**, for power users who have a higher volume of usage
+- **Completed an event regularly**, for power users who use advanced features
+
+Both represent power user behavior. When combined with the key events or actions identified earlier, we can create a cohort that matches our definition of a power user.
+
+As an example, we’ll use `uploaded_file`  as the power user event we care about. We can then either select “completed an event multiple times” or “completed an event regularly.” Because we care about the high volume of usage, and `uploaded_file` isn’t an advanced feature, we’ll choose “completed an event multiple times.” We’ll then set our usage amounts and regularity (which we can tweak later). Once done, it should look something like this:
```

Comment:
```suggestion
As an example, we’ll use `uploaded_file` as the power user event we care about. We can then either select “completed an event multiple times” or “completed an event regularly.” Because we care about the high volume of usage, and `uploaded_file` isn’t an advanced feature, we’ll choose “completed an event multiple times.” We’ll then set our usage amounts and regularity (which we can tweak later). Once done, it should look something like this:
```